bagua/torch_api/distributed.py

```diff
@@ -522,6 +575,19 @@ def forward(self, *inputs, **kwargs):
 
 
 class ModelSwitchWrapper(torch.nn.Module):
+    """
+    Transform the original module into a distributed module.
```

Comment:
ModelSwitchWrapper should be regarded as a distributed algorithm wrapper, supporting different algorithm conversion, you can give a demo

Reply:
ok

Reply:
The function description is incomplete. ModelSwitchWrapper is designed to switch distributed algorithms during training process

Like this:

model = bagua_init(DistributedAlgorithm. GradientAllReduce)
... train A epoch
model.switch_to(DistributedAlgorithm.Decentralize)
... train B epoch
model.switch_to(DistributedAlgorithm. GradientAllReduce) 
... continue training

Reply:
ok, i‘ll supplement it more complete